lib/api/entries/index.js

```diff
@@ -150,6 +150,9 @@ function configure (app, wares, ctx) {
           req.model = ctx.entries;
           break;
         default:
+          if (model && model.length > 0) {
```

Comment:
Codacy hates this function, one of the 3 remaining issues, devicestatus and treatments are the only special cases, but not special enough since they use a `created_at` field instead of the `date`/`dateString` fields found in entries